si_ps_conv_fastcorrsApp/Db/kick_limits.template

```diff
@@ -0,0 +1,24 @@
+record(ai, "$(TEST)copy-SI-$(SEC)$(DIP):PS-$(ORI):Current-$(TYPE)") {
+    field(INP, "SI-$(SEC)$(DIP):PS-$(ORI):Current-SP.$(TYPE) CP")
+    field(EGU, "A")
+    field(PREC, 3)
+    field(INP, "SI-$(SEC)$(DIP):PS-$(ORI):Current-SP.$(TYPE) CP")
+    field(FLNK, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):Kick-$(TYPE) PP")
+}
+
+record(ai, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):Kick-$(TYPE)") {
+    field(EGU, "urad")
+    field(PREC, 3)
+    field(DTYP, "Python Device")
+    field(SCAN, "5 second")
+    field(INP, "@curr2strn $(TEST) $(SEC) $(DIP) $(ORI) Current-$(TYPE)")
+    field(FLNK, "$(TEST)fwd-SI-$(SEC)$(DIP):PS-$(ORI):Kick-$(TYPE)")
```

Comment:
You're using `FLNK ... PP` for some links, but not all. Considering that it's not necessary, what do you think  about omitting `PP` for all cases?